### ISSUE TITLE
use more descriptive filenames for binaries

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#7989]
Like: `borg-linux-glibcVVV` instead of old / new / newer ...

Update 00_README.txt accordingly.

**Comment by ThomasWaldmann** [comment]
Planned text:
```
amd64 / x86_64 architecture
~~~~~~~~~~~~~~~~~~~~~~~~~~~

borg-linux-glibc236 Linux (built on Debian 12 "Bookworm" with glibc 2.36)
borg-linux-glibc231 Linux (built on Debian 11 "Bullseye" with glibc 2.31)
borg-linux-glibc228 Linux (built on Debian 10 "Buster" with glibc 2.28)
                    Note: you can also try them on other Linuxes with other glibc
                    version - as long as the glibc is compatible, they will work.

borg-macos          macOS (built on macOS Sierra 10.12 with latest macFUSE from brew)

borg-freebsd        FreeBSD 64bit (built on FreeBSD 14)

*.tgz               similar to above, but built as a directory with files,
                    not as a single self-extracting binary.
*.asc               GnuPG signatures for *
```

**Comment by ThomasWaldmann** [comment]
Automated checking, fetching, naming the binaries:
```
#!/bin/bash

mkdir -p dist/

check_and_copy () {
    echo "--- EXE $2 -----------------------------------------------"
    vagrant ssh $1 -c "/vagrant/borg/borg.exe -V"
    vagrant scp $1:/vagrant/borg/borg.exe   dist/$2
    echo "--- DIR $2 -----------------------------------------------"
    vagrant ssh $1 -c "/vagrant/borg/borg-dir/borg.exe -V"
    vagrant scp $1:/vagrant/borg/borg.tgz   dist/$2.tgz
    echo ""
}

check_and_copy buster64   borg-linux-glibc228
check_and_copy bullseye64 borg-linux-glibc231
check_and_copy bookworm64 borg-linux-glibc236

check_and_copy freebsd64  borg-freebsd

check_and_copy darwin64   borg-macos
```

**Comment by EHJ-52n** [comment]
How often do you expect these glibc specific file names to change?

**Comment by ThomasWaldmann** [comment]
@EHJ-52n For Linux, likely as often as debian does a new release.

**Comment by ThomasWaldmann** [comment]
Note: Fixed in 1.4 and master, 1.2 will not change.
